stdlib/public/Cxx/CxxDictionary.swift

```diff
@@ -10,6 +10,10 @@
 //
 //===----------------------------------------------------------------------===//
 
+/// A C++ type that represents a dictionary.
```

Comment:
it seems strange to say "A C++ type..." when this is defining a protocol.

Reply:
This seems common in the stdlib, for instance:

```
/// A type that provides sequential, iterated access to its elements.
/// ...
public protocol Sequence<Element> {
```

```
/// A type that can be initialized with an integer literal.
/// ...
public protocol ExpressibleByIntegerLiteral {
```

```
/// A type with values that support addition and subtraction.
/// ...
public protocol AdditiveArithmetic: Equatable {
```

Reply:
I see, that sounds good.